app/src/main/cpp/openxr-view-source.cc

```diff
@@ -64,6 +64,8 @@ OpenXRViewSource::Init()
     }
 
     config_views.resize(view_count);
+    std::fill(config_views.begin(), config_views.end(),
+              XrViewConfigurationView{XR_TYPE_VIEW_CONFIGURATION_VIEW});
```

Comment:
```suggestion
    config_views.resize(view_count, {XR_TYPE_VIEW_CONFIGURATION_VIEW});
```

Reply:
I applied your suggestion and removed the changes in `pch.h`.